api_v0.yml

```diff
@@ -3441,3 +3452,43 @@ paths:
           label: curl (all video articles)
           source: |
             curl https://dev.to/api/videos
+
+  /admin/users:
+    post:
+      operationId: postAdminUsersCreate
+      summary: Invites a user to the Forem instannce
```

Comment:
@Zhao-Andy I am new to the PR approval workflow and I am not sure if I am supposed to do this here... but I saw a typo `instannce` and I can't unsee 😂  What is the best way for me to file this?

Reply:
This is... a good question. 😅 I think you can edit the file directly and it'll be served with those changes. Will have to update the documentation on how to contribute to the API docs though.